running: {'--uuid': '1f262a47ce57531e83837cb069c50e21', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230425_055340', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mae', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}
cmd: python smoothness/optimizer.py -c MLP-adam -d diabetes -o smoothness -u 1f262a47ce57531e83837cb069c50e21 -m mae -n 15 -p 1 -dir /home/ryedida/bbo_challenge_starter_kit/output -b run_20230425_055340
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

computed signature: ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [151.4971925380264, 151.42763539278994, 151.47026299046811, 136.86202504264318, 75.2055865763391])
Signature errors:
                                  0             1             2         3         4       max
MLP-adam_diabetes_mae  1.919244e-08  6.198704e-08  7.624578e-09  0.000007  0.000034  0.000034
max                    1.919244e-08  6.198704e-08  7.624578e-09  0.000007  0.000034  0.000034
starting sklearn study smoothness MLP-adam diabetes mae 15 1
with data root: None
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.501265 iter 0 next_points [{'alpha': 1.942120902414143e-05, 'batch_size': 237, 'beta_1': 0.989643706932285, 'beta_2': 0.9999771828612934, 'epsilon': 1.5342447100547445e-07, 'hidden_layer_sizes': 75, 'learning_rate_init': 0.044013263040760406, 'tol': 0.0005986149254841716, 'validation_fraction': 0.8278746363400296}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.213473 value 57.660992 suggestion {'alpha': 1.942120902414143e-05, 'batch_size': 237, 'beta_1': 0.989643706932285, 'beta_2': 0.9999771828612934, 'epsilon': 1.5342447100547445e-07, 'hidden_layer_sizes': 75, 'learning_rate_init': 0.044013263040760406, 'tol': 0.0005986149254841716, 'validation_fraction': 0.8278746363400296}
observation time 0.000006, current best 57.660992 at iter 0
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.331585 iter 1 next_points [{'alpha': 0.04232953107001828, 'batch_size': 114, 'beta_1': 0.8511146810043095, 'beta_2': 0.9999241199158675, 'epsilon': 9.74235795264083e-09, 'hidden_layer_sizes': 191, 'learning_rate_init': 0.018508616666971872, 'tol': 0.0005640351935388387, 'validation_fraction': 0.8963382316879168}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.487439 value 50.675022 suggestion {'alpha': 0.04232953107001828, 'batch_size': 114, 'beta_1': 0.8511146810043095, 'beta_2': 0.9999241199158675, 'epsilon': 9.74235795264083e-09, 'hidden_layer_sizes': 191, 'learning_rate_init': 0.018508616666971872, 'tol': 0.0005640351935388387, 'validation_fraction': 0.8963382316879168}
observation time 0.000005, current best 50.675022 at iter 1
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.334147 iter 2 next_points [{'alpha': 8.306464273006137, 'batch_size': 110, 'beta_1': 0.8450367939060763, 'beta_2': 0.942083860882109, 'epsilon': 2.2862784798436773e-09, 'hidden_layer_sizes': 71, 'learning_rate_init': 6.138574519328324e-05, 'tol': 0.022897439555561745, 'validation_fraction': 0.5036924276570729}]
function_evaluation time 0.059534 value 151.621311 suggestion {'alpha': 8.306464273006137, 'batch_size': 110, 'beta_1': 0.8450367939060763, 'beta_2': 0.942083860882109, 'epsilon': 2.2862784798436773e-09, 'hidden_layer_sizes': 71, 'learning_rate_init': 6.138574519328324e-05, 'tol': 0.022897439555561745, 'validation_fraction': 0.5036924276570729}
observation time 0.000004, current best 50.675022 at iter 2
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.327041 iter 3 next_points [{'alpha': 0.34343656142996903, 'batch_size': 19, 'beta_1': 0.7709752151194296, 'beta_2': 0.9998725057500532, 'epsilon': 1.0147727094437055e-07, 'hidden_layer_sizes': 69, 'learning_rate_init': 0.03845085632872777, 'tol': 4.305401350990321e-05, 'validation_fraction': 0.5578172122150696}]
function_evaluation time 0.628650 value 44.052928 suggestion {'alpha': 0.34343656142996903, 'batch_size': 19, 'beta_1': 0.7709752151194296, 'beta_2': 0.9998725057500532, 'epsilon': 1.0147727094437055e-07, 'hidden_layer_sizes': 69, 'learning_rate_init': 0.03845085632872777, 'tol': 4.305401350990321e-05, 'validation_fraction': 0.5578172122150696}
observation time 0.000006, current best 44.052928 at iter 3
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.336273 iter 4 next_points [{'alpha': 0.08353284871938099, 'batch_size': 100, 'beta_1': 0.9862560485503802, 'beta_2': 0.9723103179792303, 'epsilon': 1.2152201073267135e-07, 'hidden_layer_sizes': 159, 'learning_rate_init': 0.0007558246081196497, 'tol': 3.521931807983432e-05, 'validation_fraction': 0.1065686290177287}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 1.558743 value 135.192036 suggestion {'alpha': 0.08353284871938099, 'batch_size': 100, 'beta_1': 0.9862560485503802, 'beta_2': 0.9723103179792303, 'epsilon': 1.2152201073267135e-07, 'hidden_layer_sizes': 159, 'learning_rate_init': 0.0007558246081196497, 'tol': 3.521931807983432e-05, 'validation_fraction': 0.1065686290177287}
observation time 0.000005, current best 44.052928 at iter 4
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.336823 iter 5 next_points [{'alpha': 0.1289458336907792, 'batch_size': 112, 'beta_1': 0.9783717349664482, 'beta_2': 0.9996700471268788, 'epsilon': 6.274128987028958e-07, 'hidden_layer_sizes': 181, 'learning_rate_init': 0.002479215446170911, 'tol': 1.4951193845830766e-05, 'validation_fraction': 0.648666799688171}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.905895 value 124.079267 suggestion {'alpha': 0.1289458336907792, 'batch_size': 112, 'beta_1': 0.9783717349664482, 'beta_2': 0.9996700471268788, 'epsilon': 6.274128987028958e-07, 'hidden_layer_sizes': 181, 'learning_rate_init': 0.002479215446170911, 'tol': 1.4951193845830766e-05, 'validation_fraction': 0.648666799688171}
observation time 0.000004, current best 44.052928 at iter 5
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.325790 iter 6 next_points [{'alpha': 0.42307945542726655, 'batch_size': 134, 'beta_1': 0.7197604650803854, 'beta_2': 0.9973947271461217, 'epsilon': 8.534933611936865e-08, 'hidden_layer_sizes': 123, 'learning_rate_init': 0.00297325059106452, 'tol': 0.010130565854109506, 'validation_fraction': 0.6450988228796508}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.055590 value 150.921542 suggestion {'alpha': 0.42307945542726655, 'batch_size': 134, 'beta_1': 0.7197604650803854, 'beta_2': 0.9973947271461217, 'epsilon': 8.534933611936865e-08, 'hidden_layer_sizes': 123, 'learning_rate_init': 0.00297325059106452, 'tol': 0.010130565854109506, 'validation_fraction': 0.6450988228796508}
observation time 0.000005, current best 44.052928 at iter 6
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.461570 iter 7 next_points [{'alpha': 0.003249018917316325, 'batch_size': 122, 'beta_1': 0.7254407639960867, 'beta_2': 0.9993829886650193, 'epsilon': 2.9976887467009293e-07, 'hidden_layer_sizes': 72, 'learning_rate_init': 0.0018976646027929135, 'tol': 0.0002065481761801782, 'validation_fraction': 0.7506809674146045}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.620603 value 138.987234 suggestion {'alpha': 0.003249018917316325, 'batch_size': 122, 'beta_1': 0.7254407639960867, 'beta_2': 0.9993829886650193, 'epsilon': 2.9976887467009293e-07, 'hidden_layer_sizes': 72, 'learning_rate_init': 0.0018976646027929135, 'tol': 0.0002065481761801782, 'validation_fraction': 0.7506809674146045}
observation time 0.000006, current best 44.052928 at iter 7
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.333088 iter 8 next_points [{'alpha': 6.227761526132728e-05, 'batch_size': 130, 'beta_1': 0.8245023454382089, 'beta_2': 0.9999936785584098, 'epsilon': 4.958919914367858e-08, 'hidden_layer_sizes': 198, 'learning_rate_init': 0.00020907103506242556, 'tol': 2.6373835092247272e-05, 'validation_fraction': 0.8412152952316947}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.807323 value 150.805651 suggestion {'alpha': 6.227761526132728e-05, 'batch_size': 130, 'beta_1': 0.8245023454382089, 'beta_2': 0.9999936785584098, 'epsilon': 4.958919914367858e-08, 'hidden_layer_sizes': 198, 'learning_rate_init': 0.00020907103506242556, 'tol': 2.6373835092247272e-05, 'validation_fraction': 0.8412152952316947}
observation time 0.000005, current best 44.052928 at iter 8
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.345864 iter 9 next_points [{'alpha': 0.00044839466771849407, 'batch_size': 149, 'beta_1': 0.6455473463299223, 'beta_2': 0.9999670973866934, 'epsilon': 1.3486485780309912e-07, 'hidden_layer_sizes': 71, 'learning_rate_init': 0.058114583284285155, 'tol': 0.029846675457770578, 'validation_fraction': 0.4733661850867242}]
function_evaluation time 0.146507 value 50.257312 suggestion {'alpha': 0.00044839466771849407, 'batch_size': 149, 'beta_1': 0.6455473463299223, 'beta_2': 0.9999670973866934, 'epsilon': 1.3486485780309912e-07, 'hidden_layer_sizes': 71, 'learning_rate_init': 0.058114583284285155, 'tol': 0.029846675457770578, 'validation_fraction': 0.4733661850867242}
observation time 0.000006, current best 44.052928 at iter 9
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.326067 iter 10 next_points [{'alpha': 2.4522158103878966, 'batch_size': 247, 'beta_1': 0.9663851572168525, 'beta_2': 0.9990862770987374, 'epsilon': 2.129633047824954e-09, 'hidden_layer_sizes': 188, 'learning_rate_init': 0.00015258401825074544, 'tol': 0.0052977756820673065, 'validation_fraction': 0.7207421103688098}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.061833 value 151.527107 suggestion {'alpha': 2.4522158103878966, 'batch_size': 247, 'beta_1': 0.9663851572168525, 'beta_2': 0.9990862770987374, 'epsilon': 2.129633047824954e-09, 'hidden_layer_sizes': 188, 'learning_rate_init': 0.00015258401825074544, 'tol': 0.0052977756820673065, 'validation_fraction': 0.7207421103688098}
observation time 0.000005, current best 44.052928 at iter 10
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.335783 iter 11 next_points [{'alpha': 0.0001901178359922289, 'batch_size': 71, 'beta_1': 0.8894502452968145, 'beta_2': 0.9922136202068154, 'epsilon': 3.704818703782769e-07, 'hidden_layer_sizes': 62, 'learning_rate_init': 9.68758511891019e-05, 'tol': 0.006053992613054659, 'validation_fraction': 0.38926642563488956}]
function_evaluation time 0.050841 value 151.486427 suggestion {'alpha': 0.0001901178359922289, 'batch_size': 71, 'beta_1': 0.8894502452968145, 'beta_2': 0.9922136202068154, 'epsilon': 3.704818703782769e-07, 'hidden_layer_sizes': 62, 'learning_rate_init': 9.68758511891019e-05, 'tol': 0.006053992613054659, 'validation_fraction': 0.38926642563488956}
observation time 0.000010, current best 44.052928 at iter 11
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.323012 iter 12 next_points [{'alpha': 5.9144846070401424, 'batch_size': 218, 'beta_1': 0.8617287127738743, 'beta_2': 0.9999683115675676, 'epsilon': 2.0593590549485722e-08, 'hidden_layer_sizes': 156, 'learning_rate_init': 0.010997505995835034, 'tol': 0.00025129502354992564, 'validation_fraction': 0.6211268635369129}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.823439 value 47.530255 suggestion {'alpha': 5.9144846070401424, 'batch_size': 218, 'beta_1': 0.8617287127738743, 'beta_2': 0.9999683115675676, 'epsilon': 2.0593590549485722e-08, 'hidden_layer_sizes': 156, 'learning_rate_init': 0.010997505995835034, 'tol': 0.00025129502354992564, 'validation_fraction': 0.6211268635369129}
observation time 0.000005, current best 44.052928 at iter 12
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.334888 iter 13 next_points [{'alpha': 0.18771971467965357, 'batch_size': 155, 'beta_1': 0.9600964478259977, 'beta_2': 0.9999988946056295, 'epsilon': 1.0425461463855647e-08, 'hidden_layer_sizes': 122, 'learning_rate_init': 0.008008953983735887, 'tol': 0.08942951294470791, 'validation_fraction': 0.27664029964646114}]
function_evaluation time 0.076629 value 147.952148 suggestion {'alpha': 0.18771971467965357, 'batch_size': 155, 'beta_1': 0.9600964478259977, 'beta_2': 0.9999988946056295, 'epsilon': 1.0425461463855647e-08, 'hidden_layer_sizes': 122, 'learning_rate_init': 0.008008953983735887, 'tol': 0.08942951294470791, 'validation_fraction': 0.27664029964646114}
observation time 0.000005, current best 44.052928 at iter 13
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 105, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 48, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.328529 iter 14 next_points [{'alpha': 1.5393206270198608e-05, 'batch_size': 42, 'beta_1': 0.9874984290387872, 'beta_2': 0.999983197655254, 'epsilon': 2.234883006900313e-08, 'hidden_layer_sizes': 100, 'learning_rate_init': 0.00027857698025577455, 'tol': 0.0018467897470747332, 'validation_fraction': 0.1601590658179192}]
function_evaluation time 0.082359 value 151.233279 suggestion {'alpha': 1.5393206270198608e-05, 'batch_size': 42, 'beta_1': 0.9874984290387872, 'beta_2': 0.999983197655254, 'epsilon': 2.234883006900313e-08, 'hidden_layer_sizes': 100, 'learning_rate_init': 0.00027857698025577455, 'tol': 0.0018467897470747332, 'validation_fraction': 0.1601590658179192}
observation time 0.000006, current best 44.052928 at iter 14
saving meta data: {'args': {'--uuid': '1f262a47ce57531e83837cb069c50e21', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230425_055340', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mae', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}, 'signature': ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [151.4971925380264, 151.42763539278994, 151.47026299046811, 136.86202504264318, 75.2055865763391])}
saving results
saving timing
saving suggest log
done
